src/client.ts

```diff
@@ -243,6 +243,9 @@ export class SubscriptionClient {
       this.eventEmitter.emit(isReconnect ? 'reconnect' : 'connect');
       this.reconnecting = false;
       this.backoff.reset();
+      // Send INIT message, no need to wait for connection to success (reduce roundtrips)
```

Comment:
If I isolate this MR to just moving where the INIT message is fired I'm not seeing any test failures.